### PR TITLE
Wl 25 hero banner fix

### PR DESCRIPTION
**Pull request by jmawloo**

### CODE REVIEW

**Comment by willakong** [comment]
It definitely looks better but around the ~1000 pixel width benchmark with a height of less than 500 pixels, both Chrome and Firefox show the "Join Us" button overlapping the yellow strip.

image
This particular image is from Chrome with dimensions 1087 x 502. 

This may or may not be an issue depending on what thresholds you are fixing it for. Other dimension combinations (pixel width of over 1500 and height of over 500 pixels) work fine and the vertical alignment of the images are spaced apart nicely.

File: waterloop-site/src/components/Hero/Landing.tsx [hunk 1]
```diff
@@ -10,41 +10,38 @@ type MyProps = { description: string };
...
-      <div className="landingContainer">
-        <div className="landingContent">
-          {/* waterloop svg */}
-          <div className="landingWaterloopContainer">
-            <img
-              className="landingWaterloop"
-              src={WaterloopImg}
-              alt="waterloop logo"
-            />
+      // ladningAligner nests background image in another div element to provide proper padding between the image and the bottom.
```
Comment (Quinn1876) [inline waterloop-site/src/components/Hero/Landing.tsx:13]: spelling ladningAligner -> landingAligner

File: waterloop-site/src/components/Hero/Landing.tsx [hunk 2]
```diff
@@ -10,41 +10,38 @@ type MyProps = { description: string };
...
+              />
+            </div>
+            <div className="landingDescriptionContainer">
+              <h3>{this.props.description}</h3>
+            </div>
+            <Button
+              backgroundColor="yellow"
+              textColor="black"
+              text="JOIN US"
+              onClick={() => window.open("recruitment")}
```
Comment (Quinn1876) [inline waterloop-site/src/components/Hero/Landing.tsx:34]: Currently opens in a new tab, is this the intended outcome?

File: waterloop-site/src/pages/Home.tsx [hunk 3]
```diff
@@ -6,6 +6,8 @@ class Home extends React.Component {
         return (
             <div>
                 <Hero description="A faster look on the future."></Hero>
+                {/* Remove bottom div once done with testing */}
```
Comment (Quinn1876) [inline waterloop-site/src/pages/Home.tsx:9]: Should this be removed then?

File: waterloop-site/src/theme/global.scss [hunk 4]
```diff
@@ -492,29 +509,33 @@ h1 {
...
-  
-  .generalDownArrow {
-    /* type img */
-    width: 27px;
-    height: 11px;
-  }
+}
 
+.landingAligner {
+  // height: 100vh;
```
Comment (Quinn1876) [inline waterloop-site/src/theme/global.scss:515]: please remove commented out section

File: waterloop-site/src/theme/global.scss [hunk 5]
```diff
@@ -492,29 +509,33 @@ h1 {
...
 .landingContainer {
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
+  justify-content: stretch;
   width: 100%;
-  height: 100vh;
+  height: calc(100vh - 60px); // To adjust the height position of the background image.
+  min-height: calc(100vw * 0.2 + 300px); // Turns out you just need a bit of linear algebra ;)
```
Comment (Quinn1876) [inline waterloop-site/src/theme/global.scss:527]: correct me if I'm wrong, but `100vw * 0.2` should just be `20vw`

File: waterloop-site/src/theme/global.scss [hunk 6]
```diff
@@ -566,41 +574,34 @@ h1 {
...
 }
 
-@media (max-width: 420px) {
+@media (max-width: 500px) {
+  .landingAligner {
+    min-height: 100vw;
+  }
+
+  .landingContainer {
+    background-image: none!important;
```
Comment (Quinn1876) [inline waterloop-site/src/theme/global.scss:590]: is there some way to do this without `!important`?  `!important` is an anti-pattern and should be avoided whenever possible.

File: waterloop-site/src/components/Hero/Landing.tsx (hunk 2 above)
Comment (jmawloo) [reply waterloop-site/src/components/Hero/Landing.tsx:34]: @jxrin should we keep it in a new tab?

**Comment by jxrin** [comment]
Hallo closing this bc we were able to fix it :D thank you!

**Pull request closed without merging**